### PR TITLE
Update docs for Polywrap Origin

### PR DESCRIPTION
**Pull request by krisbitney**
* Updates documentation to be accurate for Polywrap Origin 0.1.0
* Updates snippets to work with Polywrap Origin 0.1.0
* Rebrands "Web3Api" to "Polywrap"

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -34,9 +34,8 @@ module.exports = {
             'quick-start/create-wasm-wrappers/project-setup',
             'quick-start/create-wasm-wrappers/project-folder',
             'quick-start/create-wasm-wrappers/build-deploy-test',
-            'quick-start/create-wasm-wrappers/mutation-functions',
-            'quick-start/create-wasm-wrappers/adding-query-functions',
-            'quick-start/create-wasm-wrappers/writing-tests-with-recipes-file',
+            'quick-start/create-wasm-wrappers/adding-new-functions',
```
[the remark this answers is not in the document]
Comment (krisbitney) [reply sidebars.js:37]: see new PR https://github.com/polywrap/documentation/pull/186

File: snippets/js/create-client-instance/src/client-js-reference.ts [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 // ...And then you'll be able to use the `query`
-// method to send GraphQL requests to any Web3API
+// method to send GraphQL requests to any Polywrap
 // that's located at the specified URI.
 const result = await client.query({
   uri: 'api.example.eth',
-  query: `query {
+  query: `module {
     doSomething(
       variable: $variable
```
[the remark this answers is not in the document]
Comment (krisbitney) [reply snippets/js/create-client-instance/src/client-js-reference.ts:17]: fixed by Jordan

File: snippets/js/create-client-instance/src/get-metadata.ts [hunk 3]
```diff
@@ -1,10 +1,12 @@
-import { MetaManifest, Web3ApiClient } from '@web3api/client-js';
+import { PolywrapClient } from '@polywrap/client-js';
+import { AnyMetaManifest as MetaManifest } from '@polywrap/polywrap-manifest-types-js';
 
 export async function getMetaManifest(): Promise<MetaManifest> {
-  const client = new Web3ApiClient();
-  const uri = "ens/api.helloworld.web3api.eth";
+  const client = new PolywrapClient();
+  const uri = "ens/api.helloworld.polywrap.eth";
```
[the remark this answers is not in the document]
Comment (krisbitney) [reply snippets/js/create-client-instance/src/get-metadata.ts:6]: I believe we had a hello world app hosted at `helloworld.web3api.eth`, and the wrapper was at `api.helloworld.web3api.eth`. How would you suggest we change it? Maybe have `app.helloworld.polywrap.eth`? I don't have access to our ENS domain to make a change but we can create an issue.

File: src/docs/concepts/what-is-polywrap.md [hunk 4]
```diff
@@ -6,47 +6,43 @@ title: What is Polywrap?
...
-## What's inside a Polywrap wrapper?
+## What's inside a Polywrap Wasm wrapper?
 
-A Polywrap "wrapper" consists of the following files:
+A Polywrap Wasm "wrapper" consists of the following files:
 
-- **Query and mutation wasm modules** containing the protocol's business logic functions (e.g. Uniswap's swap functions)
-- **GraphQL** schema to provide types and parameters for the query and mutation functions
+- **A Wasm module** containing the protocol's business logic functions (e.g. Uniswap's swap functions)
+- **GraphQL** schema to provide types and parameters for the module functions
```
[the remark this answers is not in the document]
Comment (krisbitney) [reply src/docs/concepts/what-is-polywrap.md:21]: The schema is still included in the build, along with the meta manifest, so I think this is actually still accurate. How would you suggest changing it?

File: src/docs/concepts/what-is-polywrap.md [hunk 5]
```diff
@@ -6,47 +6,43 @@ title: What is Polywrap?
...
 - **Manifest** files that orchestrate the wrapper
 
 ## Where is it deployed?
 
 We currently support deploying wrappers to decentralized endpoints, [Ethereum Name Service (ENS)](https://ens.domains/), a decentralized Ethereum-based naming system and [InterPlanetary File System (IPFS)](https://ipfs.io/), a distributed P2P file system. For an example of a deployed wrapper, take a look at the ENS domain below:
 
-[ENS: Uniswap Polywrapper](https://app.ens.domains/name/v2.uniswap.web3api.eth)
+[ENS: Uniswap Polywrap wrapper](https://app.ens.domains/name/v2.uniswap.web3api.eth)
```
[the remark this answers is not in the document]
Comment (krisbitney) [reply src/docs/concepts/what-is-polywrap.md:28]: Jordan removed them

File: src/docs/demos/uniswapv3/intro.md [hunk 6]
```diff
@@ -3,15 +3,19 @@ id: intro
...
+:::caution
+This demo was made with a deprecated version of Polywrap!
+:::
+
 ## Overview
 
 Welcome to the Uniswap v3 Polywrap documentation!
 
-The Uniswap Polywrapper is written in [AssemblyScript](https://www.assemblyscript.org/), and like the official Uniswap SDK, it has a robust test suite and performs arbitrary precision arithmetic. The Uniswap Polywrapper business logic will be deployed on a decentralized endpoint, like IPFS.
+The Uniswap Polywrap wrapper is written in [AssemblyScript](https://www.assemblyscript.org/), and like the official Uniswap SDK, it has a robust test suite and performs arbitrary precision arithmetic. The Uniswap Polywrap wrapper business logic will be deployed on a decentralized endpoint, like IPFS.
```
[the remark this answers is not in the document]
Comment (krisbitney) [reply src/docs/demos/uniswapv3/intro.md:14]: Fixed, but also Jordan removed both uniswap demos

File: src/docs/demos/uniswapv3/intro.md [hunk 7]
```diff
@@ -3,15 +3,19 @@ id: intro
...
 
-Our first Polywrap client is for JavaScript(`@web3api/client-js`) and it can run in any environment that can execute JavaScript. We also have a working Rust implementation. In the future, we'll have Polywrap clients for other environments (Python, Go, and more).
+Our first Polywrap client is for JavaScript(`@polywrap/client-js`) and it can run in any environment that can execute JavaScript. We also have a working Rust implementation. In the future, we'll have Polywrap clients for other environments (Python, Go, and more).
 
-Developers integrating the Uniswap wrapper into their app would use [GraphQL](https://graphql.org/) queries to execute functions provided by the Uniswap wrapper. This documentation shows you which **query** and **mutation** functions are made available by the Uniswap wrapper and how to use them.
+Developers integrating the Uniswap Wasm wrapper into their app would use [GraphQL](https://graphql.org/) queries to execute functions provided by the Uniswap wrapper. This documentation shows you which functions are made available by the Uniswap wrapper and how to use them.
```
[the remark this answers is not in the document]
Comment (krisbitney) [reply src/docs/demos/uniswapv3/intro.md:18]: Fixed, but also Jordan removed both uniswap demos

File: src/docs/quick-start/create-js-app/create-client-instance.md [hunk 8]
```diff
@@ -3,13 +3,13 @@ id: 'create-client-instance'
...
 
-Once the Polywrap JS client has been installed, the next step is to create a `Web3ApiClient` instance:
+Once the Polywrap JS client has been installed, the next step is to create a `PolywrapClient` instance:
 
 ```typescript
 $snippet: js-create-client-instance
 ```
 
-At this point, you can already send queries to Polywraps. In the simple example below, we send one to the "hello world" Polywrap.
+At this point, you can already send queries to wrappers. In the simple example below, we send one to the "hello world" wrapper.
```
[the remark this answers is not in the document]
Comment (krisbitney) [reply src/docs/quick-start/create-js-app/create-client-instance.md:12]: fixed

File: src/docs/quick-start/create-plugin-wrappers/create-js-plugin.md [hunk 9]
```diff
@@ -50,41 +50,27 @@ Let's ensure all of your project's dependencies are installed. From inside your
...
-### **`src/index.ts`**
-The entry file simply exports all necessary types from the codegen directory. This file shouldn't need to be modified, unless you'd like to override default behaviour (with caution).
-
-### **`src/query` + `src/mutation`**
-Query & Mutation are 2 separate "modules" that exist within the plugin.
+### **`polywrap.plugin.yaml`**
+The `polywrap.plugin.yaml` manifest file describes the layout of a plugin.
 
 ### **`schema.graphql`**
+Each wrapper has a schema. This schema defines the wrapper's: dependencies, methods, and custom types. In short, it's an interface describing how to use the wrapper.
```
[the remark this answers is not in the document]
Comment (krisbitney) [reply src/docs/quick-start/create-plugin-wrappers/create-js-plugin.md:63]: fixed

File: snippets/js/create-client-instance/src/get-metadata.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (Niraj-Kamdar) [reply snippets/js/create-client-instance/src/get-metadata.ts:6]: Yes, I like it, let's keep wrapper URI short.

**Pull request merged**